scripts/flash_device.py

```diff
@@ -0,0 +1,82 @@
+import subprocess32
+import time
+from argparse import ArgumentParser
+import os
+import get_latest_build
+
+
+def connect_device():
```

Comment:
You're right, I thought it was in the Toolkit, not available here. I'll change it.